### ISSUE TITLE
  warn("Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work", RuntimeWarning)

### ISSUE THREAD

**Issue by dussypestroyer** [n3d1117/chatgpt-telegram-bot#202]
Hello. I am an inexperienced user. there is such an error when starting the bot and the bot is running slowly. ffmpeg installed

**Comment by dussypestroyer** [comment]
upd.added path to PATH but it didn't help
![изображение](https://user-images.githubusercontent.com/99405284/230740178-da7cc72a-4611-4dc3-af40-2f9c1d727a27.png)

**Comment by serverworm** [comment]
tell me how to solve this problem in code, and not using exe files

**Comment by MaximKoles** [comment]
sudo apt-get install ffmpeg

**Comment by MaximKoles** [comment]
`sudo apt-get install ffmpeg`

**Comment by dussypestroyer** [comment]
> `sudo apt-get install ffmpeg`

maybe a command for powershell?

**Comment by dussypestroyer** [comment]
I fixed it myself. what was the original problem. when I went to the "additional properties of the system" every time there was a path to ffmpeg. I deleted it and put my own, which was most likely a mistake
![изображение](https://user-images.githubusercontent.com/99405284/231986999-c0a4e1aa-caf9-461d-a826-7e60e6dc6207.png)
but I had no idea about it and tried to install ffmpeg via powershell. command for windows worked for me like this
winget install ffmpeg
after installation via powershell, the bot worked just as slowly and an error was displayed, and then I threw the files exactly along the path to PATH, where it was written in PATH and everything worked if there are questions, I will help in any way I can
